workflows/funcscan.nf

```diff
@@ -77,13 +121,123 @@ workflow FUNCSCAN {
     )
     ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
 
-    //
-    // MODULE: Run FastQC
-    //
-    FASTQC (
-        INPUT_CHECK.out.reads
-    )
-    ch_versions = ch_versions.mix(FASTQC.out.versions.first())
+    // Some tools require uncompressed input
+    INPUT_CHECK.out.contigs
```

Comment:
Missing input check versions mixing  I think?

Reply:
Nope, it is present above (l. 122).

Reply:
Oh yeah, I must have been going a bit loopy at this point :laughing: